### PR TITLE
Refactoring gpcheckcat - remove global variable and update nomenclature

### PR DESCRIPTION
**Pull request by royc1**
Update nomenclature in gpcheckcat - rename 'test' to 'check'

- This is is to be more consistent and to avoid confusion with unit/integration tests in the future.

- We purposely did not change 'test' in stdout because integration tests for other parts of the system that call gpcheckcat will fail if our output changes.  A future commit may change 'test' to 'check' in stdout in addition to fixing the aforementioned integration tests.

Authors: Chumki Roy, Stephen Wu

### CODE REVIEW

File: gpMgmt/bin/lib/gpcheckcat [hunk 1]
```diff
@@ -2168,6 +2153,7 @@ def checkOwners():
             for row in rows:
                 fixupowners(row['nspname'], row['relname'], row['rolname'],
                             row['master_rolname'])
+        return len(rows) == 0
```
Comment (chrishajas) [inline gpMgmt/bin/lib/gpcheckcat]: Do you want to return here?

File: gpMgmt/bin/lib/gpcheckcat [hunk 2]
```diff
@@ -12,9 +12,9 @@ Usage: gpcheckcat [<option>] [dbname]
     -A         : all databases
     -S option  : shared table options (none, only)
     -O         : Online 
-    -l         : list all tests
-    -R test    : run this particular test
-    -C catname : run cross consistency, FK and ACL tests for this catalog table
+    -l         : list all checks
```
Comment (Chibin) [inline gpMgmt/bin/lib/gpcheckcat]: This will require a doc update.

File: gpMgmt/bin/lib/gpcheckcat [hunk 3]
```diff
@@ -2213,7 +2198,7 @@ def checkOwners():
                        row['master_rolname']))
             if len(rows) > 100:
                 logger.error("...")
-
+        return len(rows) == 0
```
Comment (chrishajas) [inline gpMgmt/bin/lib/gpcheckcat]: Same thing here, there's code below that may need to be executed.

**Comment by ivannovick** [comment]
great idea to make gpcheckcat more testable

**Comment by shwu** [comment]
This PR is now covered by #620; closing this one and merging the other one instead.

**Pull request closed without merging**